cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java

```diff
@@ -388,6 +391,21 @@ private Set<Employment> extractEmployments() {
                    .collect(Collectors.toSet());
     }
 
+    private Boolean extractVerified() {
+        var nviVerified = Optional.ofNullable(extractNvi())
```

Comment:
Not sure if I understand why this part is necessary:
```
.map(PersonNvi::verifiedBy)
.map(PersonSummary::id)
.map(UriUtils::extractLastPathElement)
.filter(Utils::isPositiveInteger)
```

Reply:
Would it be enough with `var nviVerified = Optional.ofNullable(extractNvi()).isPresent();`?
(I checked out your code and tried this, test was still green)

Reply:
I just wanted to make sure we actually have data and not just empty objects, but maybe im overthinking it